companies/slovnaftbajk.json

```diff
@@ -0,0 +1,19 @@
+{
+    "slug": "slovnaftbajk",
+    "relevant-countries": [
+        "sk"
+    ],
+    "categories": [
+        "travel"
+    ],
+    "name": "Slovnaft Mobility Services, s. r. o.",
+    "address": "Vlčie hrdlo 1\n82412 Bratislava\nSlovakia",
+    "phone": "+421 2 5950 5950",
+    "email": "info@slovnaftbajk.sk",
```

Comment:
As the email is not specific to data protection, we can't set the `suggested-transport-medium` to `email`. You can just remove that line.

Reply:
removed